packages/storybook/src/nimble/table-column/anchor/table-column-anchor.stories.ts

```diff
@@ -40,41 +41,66 @@ export default metadata;
 const simpleData = [
     {
         firstName: 'Ralph',
+        firstNameSortIndex: 3,
         lastName: 'Wiggum',
+        lastNameSortIndex: 3,
+        fullName: 'Ralph Wiggum',
```

Comment:
What about adding a field like address instead? Like Ralph is at `732 Evergreen Terrace` and address is something one could argue is more useful to sort by the street name than alphanumeric